client/signup/steps/website-content/section-generator.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+	AccordionSectionProps,
+	SectionGeneratorReturnType,
+} from 'calypso/signup/accordion-form/types';
+import { WebsiteContent } from 'calypso/state/signup/steps/website-content/schema';
+import { PageDetails } from './page-details';
+
+const PAGE_TITLES: Record< string, string > = { Home: 'Homepage' };
```

Comment:
GOod point I will rethink this :)

Reply:
Thanks for this suggestion and I have dynamically generated the initial pages with translated titles. I experimented with the dynamically created page based on the vertical and it is easily doable. 
But I thought I could tackle it with the next immediate PR since there might be changes related to options that might clash here.